### PR TITLE
Scheduled Profiler - Bugfixes from testing

### PR DESCRIPTION
**Pull request by BazookaMusic**
#### Summary
Some bugs were identified on further testing of the feature. These are:
1) While non-admin users cannot see other users' profiles and schedules, they can create them as insert wasn't guarded on the page.
2) The description is not correctly validated to be non-empty
3) On the performance profiles page, we incorrectly include "(ms)" in the caption although the duration type is formatted with a textual form of the duration.

#### Work Item(s)
Fixes [AB#544034](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/544034)

### CODE REVIEW

File: src/System Application/App/Performance Profiler/src/PerfProfilerScheduleCard.Page.al [hunk 1]
```diff
@@ -223,10 +233,17 @@ page 1932 "Perf. Profiler Schedule Card"
         MaxRetentionPeriod: Duration;
         NoRetentionPolicySetupErr: Label 'No retention policy setup found for the performance profiles table.';
         CreateRetentionPolicySetupTxt: Label 'Create a retention policy setup';
+        NotEmptyDescriptionErr: Label 'The description must be filled in.';
```
Comment (darjoo) [inline src/System Application/App/Performance Profiler/src/PerfProfilerScheduleCard.Page.al]: The variable name dis different from what the label is. It sounds like the description is not empty. When the label is a message saying that is must be filled in.
I would rename to something like `DescriptionNotFilledErr` or `EmptyDescriptionErr`.

File: src/System Application/App/Performance Profiler/src/ScheduledPerfProfilerImpl.Codeunit.al [hunk 2]
```diff
@@ -220,4 +231,5 @@ codeunit 1932 "Scheduled Perf. Profiler Impl."
         ProfileCannotBeInThePastErr: Label 'A schedule cannot be set to run in the past.';
         ScheduleDurationCannotExceedRetentionPeriodErr: Label 'The performance profile schedule duration cannot exceed the retention period.';
         ScheduleEndTimeCannotBeEmptyErr: Label 'The performance profile schedule must have an end time.';
+        CannotCreateSchedulesForOtherUsersErr: Label 'Only user admins can create schedules for other users.';
```
Comment (darjoo) [inline src/System Application/App/Performance Profiler/src/ScheduledPerfProfilerImpl.Codeunit.al]: Sounds weird. `Only admin users...` maybe? Check with the design team, Jacob, as they should review labels.

Comment (darjoo) [reply src/System Application/App/Performance Profiler/src/ScheduledPerfProfilerImpl.Codeunit.al]: Additionally, is the text actually correct? It doesn't exactly require an admin user does it? A user could be given `SECURITY` (Which is what `CanManageUsersOnTenant` checks for).

**Review by darjoo — APPROVED** [review]

File: src/System Application/App/Performance Profiler/src/ScheduledPerfProfilerImpl.Codeunit.al (hunk 2 above)
Comment (BazookaMusic) [reply src/System Application/App/Performance Profiler/src/ScheduledPerfProfilerImpl.Codeunit.al]: Fixed

File: src/System Application/App/Performance Profiler/src/PerfProfilerScheduleCard.Page.al (hunk 1 above)
Comment (BazookaMusic) [reply src/System Application/App/Performance Profiler/src/PerfProfilerScheduleCard.Page.al]: Fixed

**Review by JesperSchulz — APPROVED** [review]

**Pull request merged**